news/2020-31-12-mappers-report-fall.md

```diff
@@ -0,0 +1,265 @@
+
+### guden (osu!mania)
+
+A promising upcoming name amongst both the ranked and the competitive scene is [guden](https://osu.ppy.sh/users/11626065), having gotten their first qualified just a couple of days ago, namely his set of [Camellia - Alone intelligence](https://osu.ppy.sh/beatmapsets/1235533#mania/2568385). Overall, his mapping style shows a charming blend of several notable influences, such as mint’s consistent structure, the unconventional, unique concepts of Abraxos, the melodic style of Shoegazer, and Monheim’s expressive and allocated layering. 
+
+All these influences come in very well together in his [kaitendaentai - Hizai ni Negu](https://osu.ppy.sh/beatmapsets/1304769#mania/2705462), notably in how well the barrage of notes flows that illustrates the breakcoreish, frantic drums. Other elements: how well many aspects of the music are structured; how specific chords are allocated for specific events; and how certain unusual musical moments is mapped in unusual ways, are combined in a rather well-executed manner.
+
+While they are still trying to find their own voice in the scene, it will be quite fascinating to see how guden will develop as a mapper. He is always willing to try something new and has already been making great strides in improving.
+
+### MJHs (osu!mania)
+
+[MJHs](https://osu.ppy.sh/users/2288943) is a Korean mapper who’s been in and out of the mania mapping community since late 2018, but has been getting a bit of traction as of recently within the anti-meta keymode community, primarily 5K. A lot of his maps from the start of his mapping career were pretty straightforward and simplistic to say the least, however he’s been notably trying new and innovative concepts to implement into his more newer maps that definitely make him stand out amongst a crowd of 5K mappers. And though the 5K community is very small to begin with, his dedication to the keymode is unwavering to say the least and it’s why this small, niche spot in the community is why many of his maps are starting to hold amazingly in the current 5K mapping scene, despite not having any ranked maps as of now.
+
+One of his most solidified maps in his mapping discography is [Konami Amusement (Wakita Jun) - Vide Nostalgia](https://osu.ppy.sh/beatmapsets/1197900#mania/2530522), which not only displays some excellent timing craftsmanship, but as well as heavy PR patterning whilst maintaining mapping standards that fit along with the song beautifully. The map also features a constant 3/4 timing signature that can somewhat throw players off from your normal 4/4 timing signature that many are accustomed to, making this quite a challenge to acc. The mapset as a whole resembles MJH’s current mapping style perfectly, which is why it holds such an important place in his mapping discography. 
```

Comment:
which not only displays some excellent timing craftsmanship, but ~~as well as~~  **also shows well executed,** heavy PR patterning

"acc" >> "obtain accuracy on"